models.py

```diff
@@ -62,9 +61,10 @@ def from_row(cls, row: Row) -> "Charges":
 
     @property
     def time_left(self):
+        assert self.last_accessed_at, "Charge has not been accessed yet."
```

Comment:
why is this required?

Reply:
its optional... on models so the calculation wouldnt work is none